### ISSUE TITLE
"Missing translation: XX"

### ISSUE THREAD

**Issue by t4t5** [jamesarosen/ember-i18n#240]
Not sure if this is a bug or if I'm doing something wrong.

I come from the ember-cli-i18n library and wanted to switch to this one in order to use Glimmer.

I've included the library and have all my translations in `app/locales/en/translations.js`. However, when using `{{t "something"}}`, the only thing I see is "Missing translation: something".

What's weird is that if I try to use the same translation in the compoonent.js-file (`this.get('i18n').t('something')`), it works great!

I am using Ember 1.13.1.

**Comment by jamesarosen** [comment]
Is it possible you have another `t` helper registered, possibly from another i18n library? The [tests](https://github.com/jamesarosen/ember-i18n/blob/v4.0.0-beta.3/tests/dummy/app/templates/index.hbs) for helper invocation are fairly thorough.

**Comment by t4t5** [comment]
Darn it. Finally found the issue!

I think I used this library a long time ago, because I had a Bower version of it installed and imported in my `Brocfile.js`. I then forgot about it, and later assumed that it was installed as part of the `ember install ember-i18n` process that I did a few days ago, but yeah, it was actually just an old version of this library that I forgot to remove. So that's what caused the conflict.

Thanks for the help @jamesarosen, keep up the great work! :+1:
